File: C Programming/Assignments/Assignment 6/function with no return type and no para/Assignment 1/Q3/Q3.c
//Write a program to check whether a given year is a leap year.

#include<stdio.h>

void checkLeapYear();
void main(){
	
	checkLeapYear();
	
}
	
void checkLeapYear(){

	//variable initialization
	int year = 2000;
	
	//checking for leap year
	if(year%4==0 && year%100!=0 || year%400==0)
		printf("%d is a leap year",year);
	else
		printf("%d is not an leap year",year);
		
}
	

File: FirstBit Workspace/C Programming/Assignments/java_to_c_connectivity/Assignment 1/Q3/Q3.java
//Write a program to check whether a given year is a leap year.

public class Q3 {
    public static void main(String[] args) {
        // variable initialization
        int year = 2000;

        // checking for leap year
        if (year % 4 == 0 && year % 100 != 0 || year % 400 == 0)
            System.out.println(year + " is a leap year");
        else
            System.out.println(year + " is not a leap year");
    }
}
